builder/commands/test.py

```diff
@@ -1,38 +1,46 @@
 import os
 import shutil
-import ConfigParser
 import logging
-import sys
 import argparse
 
 from cliff.command import Command
 from builder.converter.converter import convert_to_xml
-from builder.uploader.jenkins_upload import update
 
 
 class Test(Command):
-    "Spits out the generated xmls in the out folder of cwd"
-
+    """
+    Spits out the generated xmls for input yaml(s) in the configurable out folder
+    """
     log = logging.getLogger(__name__)
 
     def get_parser(self, prog_name):
         parser = argparse.ArgumentParser(description="Parser")
         parser.add_argument("yaml",
                             type=str,
+                            nargs="+",
                             help="Path to the view yaml file")
+        parser.add_argument("-o",
+                            "--out-dir",
+                            type=str,
+                            dest="out_dir",
+                            default="out",
+                            help="Path to XML output dir")
         return parser
 
     def take_action(self, parsed_args):
-        with open(os.path.join(parsed_args.yaml), 'r') as yaml_file:
-            yaml = yaml_file.read()
-            self.log.debug(yaml)
+        out_dir = parsed_args.out_dir
+        if os.path.isdir(out_dir):
+            shutil.rmtree(out_dir, ignore_errors=True)
+        os.makedirs(out_dir)
+        for yaml_filename in parsed_args.yaml:
+            with open(os.path.join(yaml_filename), 'r') as yaml_file:
+                yaml = yaml_file.read()
+                self.log.debug(yaml)
+
+            try:
+                name, xml = convert_to_xml(yaml)
+            except Exception as e:
+                raise(e)
 
-        try:
-            name, xml = convert_to_xml(yaml)
-        except Exception as e:
-            raise(e)
-        if os.path.isdir("out"):
-            shutil.rmtree("out", ignore_errors=True)
-        os.makedirs("out")
-        with open(os.path.join("out", name + ".xml"), 'w') as xml_file:
-            xml_file.write(xml)
+            with open(os.path.join(out_dir, name + ".xml"), 'w') as xml_file:
+                xml_file.write(xml)
```

Comment:
I don't actually see any point here, there is a debug log about which yaml file is being processed and the output of each view is written in xml (in output directory).

Or you were asking just to log the name of the output.xml?